apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SoftwareCheckStep.tsx

```diff
@@ -65,6 +66,10 @@ const SoftwareCheckStep = ({ isDisplayed, onComplete }: Props) => {
       return;
     }
 
+    if (devicePermissionState === "refused") {
```

Comment:
Isn't there a variable (like `SoftwareCheckStatus.refused`) instead of hardcoding "refused" ? I see that for other states below there is one.

Reply:
There is no state for refused